bot.py

```diff
@@ -177,24 +152,24 @@ async def result(ctx, *args):
             winner = team2[3:] + " wins"
             winner_icon = "https://twemoji.maxcdn.com/v/latest/72x72/32-20e3.png"
             winner_reaction_index = 1
-            
+
         description = ""
         for map_played, map_score in scores.items():
             description += map_played + ": " + map_score
-        
+
         result_embed = discord.Embed(title="**Match result**", color=0x2ecc71)
```

Comment:
Maybe change this to `Embed` given we've got it imported

Reply:
I'm planning to rewrite some of this function in another PR.